.travis.yml

```diff
@@ -3,8 +3,9 @@ os:
   - linux
   - osx
 julia:
-#  - release # 0.3 not supported yet
-  - nightly
+  - 0.4
+  - 0.5
+# - nightly
```

Comment:
instead of commenting this out, make it an allowed failure so it'll run but won't make the status red?